install/connector/docs/ReleaseNotes-JCA-2.0.html

```diff
@@ -1,130 +1,132 @@
-</blockquote>
-<hr>
-<h2>Known Problems and Workarounds</h2>
-
-<h3>Running the Jakarta Connectors TCK Against Jakarta EE CI on Windows</h3>
-<blockquote>
-<p>When configuring and running the Jakarta Connectors TCK against the Eclipse GlassFish 6 Compatible Implementation
-    (CI) on Windows, please make sure to install all related software (Java SE, the CI, the TCK, etc.) on the same drive (for example, <code>C:</code>). If you want to install on a drive other than <code>C:</code>, please be sure to change the following properties in <code>&lt;TS_HOME&gt;/bin/ts.jte:</code></p>
-
-<pre>
-s1as.applicationRoot=c:
-</pre>
-</blockquote>
-<hr>
-<p><cite><small><address>
-Copyright (c) 2013, 2020 Oracle and/or its affiliates. All rights reserved.
-</address></small><cite></p>
-</body></html>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
+    <title>Jakarta Connectors TCK Release Notes</title>
+    <!--
+    Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.    This program and the accompanying materials are made available under the    terms of the Eclipse Public License v. 2.0, which is available at    http://www.eclipse.org/legal/epl-2.0.    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,    version 2 with the GNU Classpath Exception, which is available at    https://www.gnu.org/software/classpath/license.html.    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0-->
+    <style type="text/css">
+-->
+body {  background-color: #ffffdd; color: #000000}
+pre {  background-color: #eeeeee; color: #000099; font-size: smaller}
+code {  color: #000099; font-size: smaller}
+h1 {  color: #000099}
+h2 {  color: #000099}
+h3 {  color: #000099; text-decoration: underline}
+h4 {  font-style: italic; color: #000099}
+.tabletitle {  font-style: italic; color: #ffffcc; font-size: small; font-variant: small-caps}
+.subtitle {  font-style: italic; color: #ffffcc; font-variant: small-caps}
+.emphasize  {  font-style: italic; color: #000099}
+.step  { font-weight: bold}
+.note { background-color: #fffff0; border-bottom-width: thin; border-top-width: thin; border-top-style: inset; border-bottom-style: inset; font-size: -10%;}
+th {  background-color: #eeeeee; padding-left: 2pt; padding-right: 2pt;}
+-->
+</style></head>
+  <body>
+    <div align="center">
+      <h1>Jakarta EE Connectors Specification Technology Compatibility Kit,
+        Version 2.0</h1>

```

Comment:
For Connectors TCK as well, we have updated the version to 2.0.1

Reply:
I don't intend to update the micro-version changes.